staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/subresource.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fieldmanager
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal"
+	"sigs.k8s.io/structured-merge-diff/v4/fieldpath"
+)
+
+// FindEntries returns all the managed fields which has a specific path
+func FindEntries(entries []metav1.ManagedFieldsEntry, pathParts ...interface{}) ([]metav1.ManagedFieldsEntry, error) {
+	path, err := fieldpath.MakePath(pathParts...)
```

Comment:
Can we build the path in the calling function so that the type is more obvious in the function signature (and doesn't require taking variable arguments)